src/asyncrpcoperation.cpp

```diff
@@ -27,15 +27,19 @@ std::map<OperationStatus, std::string> OperationStatusMap = {
 /**
  * Every operation instance should have a globally unique id
  */
-AsyncRPCOperation::AsyncRPCOperation() : error_code_(0), error_message_() {
+AsyncRPCOperation::AsyncRPCOperation(CWallet* const wallet) :
```

Comment:
what is the ownership model for wallets?
i think you'll want to hang onto a shared pointer here so that the wallet sticks around until you're done if someone happens to call `RemoveWallet` in the interim